src/main/java/com/gtnh/findit/service/blockfinder/ClientBlockFindService.java

```diff
@@ -25,6 +29,17 @@ public ClientBlockFindService() {
     public void handleResponse(EntityClientPlayerMP player, BlockFoundResponse response) {
         player.closeScreen();
         ParticlePosition.highlightBlocks(player.worldObj, response.getPositions());
+        lookAtTarget(player, response);
+    }
+
+    public void lookAtTarget(EntityClientPlayerMP player, BlockFoundResponse response){
```

Comment:
Consider moving this to a util class so its not duplicated between these this class and `ClientItemFindService`